src/test/java/pageObject/pages/InMailPageFactory.java

```diff
@@ -0,0 +1,51 @@
+package pageObject.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import pageObject.BaseEntities.BasePages;
+
+public class InMailPageFactory extends BasePages {
+
+    private final static By loginLogo = By.cssSelector(".personal-info-login__text");
+    private final static By menuButton = By.xpath("//a/div//img");
+    private final static By logOut = By.xpath("//ul[@class = 'menu__group']//span[text() = 'Log out']");
```

Comment:
is it the only locator you could find? searching by text is not the best option.